test/e2e/static_pods.go

```diff
@@ -32,6 +34,14 @@ func waitForPodRunningInNamespaceTimeout(c clientset.Interface, podName, namespa
 }
 
 func createStaticPod(f *framework.Framework, nodeName string, podYaml string) {
+	// FIXME; remove need to use a container runtime because its not portable
+	runCommand := func(cmd ...string) (string, error) {
+		output, err := exec.Command(cmd[0], cmd[1:]...).CombinedOutput()
+		if err != nil {
+			return "", fmt.Errorf("failed to run %q: %s (%s)", strings.Join(cmd, " "), err, output)
+		}
+		return string(output), nil
+	}
```

Comment:
so what happens with tests that use these functions? are they skipped when runtime is not docker? I see docker specifically used below. Could you find the node's IP and use scp/rsync instead? Or make provider specific methods where docker/podman use their native commands, while infra provider uses scp/rsync?

Reply:
I skipped it because i didnt want to take the time thinking about an api for this.
It didnt fit naturally in the infra provider api so i just skipped it for now.

Reply:
ya, i thought about your solution but i didnt think it fix with what the provider api is trying to do so I ignored it because of the effort required.